test/chainsaw/traces/verify-traces.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+KEPTN_NAMESPACE="keptn-system"
+RETRY_COUNT=5
+SLEEP_TIME=5
+# span name which we expect to find in the logs of otel-collector
+EXPECTED_SUBSTRINGS=(
+    Name : keptnapp-0.0.1-6b86b273
```

Comment:
Can we also add a check for the namespace of the spans here, in case we want to test more tracing scenarios later?

Reply:
There is a possibility to check the attributes of the spans

```
Attributes:
     -> keptn.deployment.app.name: Str(keptndemoapp)
     -> keptn.deployment.app.version: Str(0.0.1)
     -> keptn.deployment.app.namespace: Str(keptndemo)
```

but we cannot link the spans with the attributes, since I haven't find a way to write out the span logs in JSON (or different standardized format) to be able to parse it and check multiple information in time

Would searching for the attributes be a solution for you?

Reply:
ok thanks for the clarification - Searching for the attributes will increase confidence that the spans we find are actually the ones we expect for this particular test, and not for some other deployment from another test